src/components/TableResize.js

```diff
@@ -94,45 +94,32 @@ class TableResize extends React.Component {
   };
 
   onResizeStart = (id, e) => {
-    this.setState({ isResize: true, id, startPosition: e.clientX });
+    this.setState({ isResize: true, id });
   };
 
   onResizeMove = (id, e) => {
-    const { startPosition, isResize, resizeCoords } = this.state;
-    const fixedMinWidth = 50;
-    let cellPositions = [];
+    const { isResize, resizeCoords } = this.state;
+    const fixedMinWidth = 100;
```

Comment:
Can you explain the use of `100` here instead of `50`?

Reply:
It is arbitrary number. In my previous solution was smaller, as was adding right padding of cell. However after second thought on this as anyway could not connect this with width of cell decided to skip looping through all cells to get padding array and  decided to have it fixed, but then it needed bit more space.